articles/user-profile/normalized/auth0/index.md

```diff
@@ -2,6 +2,10 @@
 description: Auth0 normalizes common user properties in the User Profile.
 ---
 
+::: version-warning
+The user profile returned after successful authentication using an OIDC-compliant flow may differ from the Auth0 normalized profile. Please use the toggle to see the claims available on that profile.
+:::
+
```

Comment:
Move the panel under the H1. I would also like to see a different text. You say what the other page is but you don't say what this page is. What is an OIDC-compliant flow? Try to make it more clear and add a link for more info.

I would expect something like this:
"This article describes the normalized user profile, which is a protocol-agnostic representation of this user as defined by Auth0....." Describe what this is and then say that if you are looking for the other thing use the toggle. You can check out this doc https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims